src/components/Button.js

```diff
@@ -9,17 +8,23 @@ class Button extends React.Component {
   }
 
   render() {
-    const { btnValue, className } = this.props;
+    const { value, clickEvent, className } = this.props;
     return (
-      <button type="button" className={`btn ${className}`}>
-        {btnValue}
+      <button
+        type="button"
+        className={className}
+        value={value}
+        onClick={clickEvent}
```

Comment:
- [ ] At the initial state of your application, when I press the **`%`** key twice consecutively, your application crashes, displaying the error (see image), please fix this.

- [ ] After I click the `AC` button and click any operation button, e.g, `+`, `-`, or `*`, `null` is displayed on the screen, one more click of the aforementioned buttons, your application crashes, please fix this.

<img width="1055" alt="Screen Shot 2021-10-22 at 12 11 00 AM" src="https://user-images.githubusercontent.com/47994600/138373429-771f44a2-8b7d-4e2c-b8ed-c569d5f4da0f.png">